bears/c_languages/CPPLintBear.py

```diff
@@ -25,13 +25,19 @@ class CPPLintBear:
     @staticmethod
     def create_arguments(filename, file, config_file,
                          max_line_length: int=79,
+                         use_spaces: bool=True,
+                         indent_size: int=2,
                          cpplint_ignore: typed_list(str)=(),
                          cpplint_include: typed_list(str)=()):
         """
         :param max_line_length: Maximum number of characters for a line.
         :param cpplint_ignore:  List of checkers to ignore.
         :param cpplint_include: List of checkers to explicitly enable.
         """
+        if (not use_spaces):
+            raise ValueError("CPPLint requires use_spaces=True")
```

Comment:
I recently did a similar thing for `DartLintBear` (#950), there we agreed on the error message saying that the *Bear* does not support/require certain settings rather than the wrapped tool, as the user might not even know which tool they are using. So please change this to something line `CPPLintBear requires...`